### PR TITLE
Health check

### PR DESCRIPTION
**Pull request by pink-lucifer**
Greeting, this is part of change regarding #31,
Not yet adding custom endpoint, simply utilizing the health status manager in grpc-services module, and providing a service health check rpc service; 
Additionally, this only take care of start/stop, runtime status change not included yet.

please help review, thanks & regard.

### CODE REVIEW

File: grpc-spring-boot-starter/src/main/java/org/lognet/springboot/grpc/GRpcServerRunner.java [hunk 1]
```diff
@@ -12,20 +15,18 @@
...
 /**
- *  Hosts embedded gRPC server.
+ * Hosts embedded gRPC server.
  */
 @Slf4j
-public class GRpcServerRunner implements CommandLineRunner,DisposableBean  {
-
-
+public class GRpcServerRunner implements CommandLineRunner, DisposableBean {
+    private final Set<String> serviceList = new ConcurrentSet<>();
```
Comment (jvmlet) [inline grpc-spring-boot-starter/src/main/java/org/lognet/springboot/grpc/GRpcServerRunner.java]: why do you need ConcurrentSet here ? Also it's from `io.netty.util.internal` ...

Comment (pink-lucifer) [inline grpc-spring-boot-starter/src/main/java/org/lognet/springboot/grpc/GRpcServerRunner.java]: Regarding ConcurrentSet, my thought is the health status check is likely for LB ... which means governance  tool might change the status during the runtime. 
As per netty package, i think it's already included in the dependency.

Comment (jvmlet) [inline grpc-spring-boot-starter/src/main/java/org/lognet/springboot/grpc/GRpcServerRunner.java]: > As per netty package, i think it's already included in the dependency.
Yes, it's included, but it's from `internal` package, please use standard concurrent set from JDK (`ConcurrentSkipListSet`  maybe ?)

**Comment by jvmlet** [comment]
Great, thanks for the PR. So in future we will be able to expose the **http** health check  endpoint to integrate with spring cloud infra (like eureka/ect.)

**Comment by pink-lucifer** [comment]
Yep, i have made the change from netty ConcurrentSet to jdk ConcurrentSkipListSet, please help review, 

Thanks & Best Regards.

**Pull request merged**